packages/gitlab/Import/Method/SFT_getGitLabRepositories.xml

```diff
@@ -14,11 +14,16 @@ string gitLabUrl= inn.getItemById("Variable","49536CF1DED041C48488461525C93DD0")
 
 // prepare the REST Call
 string html = string.Empty;
-System.Net.ServicePointManager.ServerCertificateValidationCallback = delegate { return true; };
+//System.Net.ServicePointManager.ServerCertificateValidationCallback = delegate { return true; };

```

Comment:
Remove commented out code, unless it's here for a reason. If there's a reason, add another comment explaining it.

Reply:
Code Removed.